services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/toggles/FeatureTogglesV2Spec.groovy

```diff
@@ -112,4 +122,159 @@ class FeatureTogglesV2Spec extends HealthCheckSpecification {
         then: "Able to delete flows"
         [flow, newFlow].each { flowHelperV2.deleteFlow(it.flowId) }
     }
+
+    @Ignore("https://github.com/telstra/open-kilda/issues/2955")
+    @Tags(HARDWARE)
+    def "Flow encapsulation type is changed while auto rerouting according to 'flows_reroute_using_default_encap_type' \
+feature toogle"() {
+        given: "A switch pair which supports transit_vlan and vxlan encapsulation types"
+        def swPair = topologyHelper.getAllNeighboringSwitchPairs().find {
+            [it.src, it.dst].every { it.noviflow && !it.wb5164 } && it.paths.size() >= 2
```

Comment:
I understand what you mean, but this is an 'indirect' search. 'noviflow' switch just means that the manufacturer is 'Noviflow', it is not directly say anything about vxlan. Instead, you should look for 'COPY_FIELD' feature OR look at the switch properties allowed encapsulation type (not sure what is better atm. I guess the second option is better given #2941 is implemented)

Reply:
On the other hand, _feature_ is more of a background property available only in database which can be changed, i.e. has no 'contract', while switch props are part of API. Also, I'm wondering if we can actually build a vxlan flow on a switch, which has COPY_FIELD but doesn't have VXLAN encap type enabled. Can you check that?

Reply:
I will check it
I believe that system will not create a vxlan flow

Reply:
System looks on switch properties, not on switch features. you should do the same

Reply:
Switch can have COPY_FIELD feature, but have a turned off VXLAN encap type property. In this case this switch is considered as a non-vxlan. Your test does not account for this situation

Reply:
thx, fixed